web/src/components/Post/PostForm/PostForm.tsx

```diff
@@ -172,18 +159,57 @@ const PostForm = (props: PostFormProps) => {
           )}
 
           {postType === EPostType.ARTICLE && (
-            <div className="mt-4">
-              <Upload
-                name="coverImage"
-                multiple={false}
-                setCoverImage={({ public_id, secure_url }) =>
-                  setCoverImage({
-                    imageId: public_id,
-                    url: secure_url,
-                  })
-                }
-              />
-            </div>
+            <>
+              <div className="mt-4">
+                <Upload
+                  name="coverImage"
+                  multiple={false}
+                  setCoverImage={({ public_id, secure_url }) =>
+                    setCoverImage({
+                      imageId: public_id,
+                      url: secure_url,
+                    })
+                  }
+                />
+                <div className="my-2 flex flex-row items-center">
+                  <CheckboxField
+                    name="addPhotos"
+                    defaultValue={addPhotos}
+                    className="rw-input mb-1"
+                    errorClassName="rw-input rw-input-error"
+                    onChange={() => {
+                      setAddPhotos(addPhotos ? false : true)
+                    }}
+                    validation={{ required: true }}
+                  />
```

Comment:
What's the reason for adding this checkbox and state? If you have added photos and then you edit the post, it will be false again and you won't see the photos upload button.